dart/lib/src/event_processor/enricher/enricher_event_processor.dart

```diff
@@ -1,7 +1,7 @@
 import '../../event_processor.dart';
 import '../../sentry_options.dart';
 import 'io_enricher_event_processor.dart'
-    if (dart.library.html) 'web_enricher_event_processor.dart';
+    if (dart.library.js_interop) 'web_enricher_event_processor.dart';
```

Comment:
is it possible to keep the original code in place for consumers with an older Flutter version? Something like 
```suggestion
    if (dart.library.js_interop) 'web_enricher_event_processor.dart';
    if (dart.library.html) 'web_enricher_event_processor_legacy.dart';
```

Reply:
that works and is also done inside the poc

in your scenario example `web_enricher_event_processor_legacy.dart` would use dart:html

and `'web_enricher_event_processor.dart' would use package:web, it can't use dart:html since that would fail the compilation so we're back at the point where we have to add the web package dependency. 

Just looks to me like we can't avoid that unless I see it wrong xd